src/transformers/models/whisper/convert_openai_to_hf.py

```diff
@@ -51,6 +54,9 @@
 }
 
 
+MEL_FILTERS_URL = "https://raw.githubusercontent.com/openai/whisper/main/whisper/assets/mel_filters.npz"
```

Comment:
that's alright but would rather have our processor correctly create them, let's try to make sure this is the case